README.md

```diff
@@ -15,9 +14,16 @@ def deps do
 end
 ```
 
-Documentation can be generated with [ExDoc](https://github.com/elixir-lang/ex_doc)
-and published on [HexDocs](https://hexdocs.pm). Once published, the docs can
-be found at [https://hexdocs.pm/kadena](https://hexdocs.pm/kadena).
+## Configuration
+```elixir
+config :kadena, network: :test # Default is `:test`. To use the public network, set it to `:public`
+
```

Comment:
We have not yet defined how to manage Kadena's network configuration. This is an assumption